life50x50/Form1.cs

```diff
@@ -0,0 +1,206 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+
+namespace life50x50
+{
+    public partial class Form1 : Form
+    {
+        private bool[,] gameboard = new bool[50, 50];
+        private bool gamestarted = false;
+        private int stage;
+        private int changes;
+        private int totalAlive;
+        private Timer timer = new Timer();
+
+
+        Pen myPen = new Pen(Color.Gray);
+        Brush myBrush = new SolidBrush(Color.Black);
+        Graphics g = null;
+
+        public Form1()
+        {
+            InitializeComponent();
+            timer.Tick += new EventHandler(TimerEventProcessor);
+            timer.Interval = 1000;
+            menuStrip1.Items[1].Visible = false;
+            prepgame();
+        }
+
+        private void prepgame()
+        {

```

Comment:
Метод подготовки игры не является обработчиком события, поэтому лучше к нему применить обычный стиль именования (а не тот, который предлагает конвенция Microsoft для оформления событий), то есть хотя бы называть его с большой буквы. Желательно также, чтобы название метода начиналось с глагола в императиве. Ну вижу смысла сокращать слова. И также лучше использовать стиль camelCase.
```suggestion
            PrepareGame();
        }

        private void PrepareGame()
        {

```